src/api/revealTreeItem.ts

```diff
@@ -10,7 +10,7 @@ export async function revealTreeItem(resourceId: string): Promise<void> {
     return await callWithTelemetryAndErrorHandling('api.revealTreeItem', async (context: IActionContext) => {
         const node: AzExtTreeItem | undefined = await ext.tree.findTreeItem(resourceId, { ...context, loadAll: true });
         if (node) {
-            await ext.treeView.reveal(node, { select: true, focus: true, expand: true });
+            await ext.treeView.reveal(node, { select: true, focus: true, expand: false });
```

Comment:
Why change this behavior?

Reply:
😄  I didn't like when it auto expanded it. Might be good to be able to pass in these options to the revealTreeItem command.